career-readiness.md

```diff
@@ -6,16 +6,18 @@ The LEARN program focuses on readying our students for a career in technology. T
 
 Deploying a project is an exciting, yet often challenging process. There are usually small errors that cause deployments to fail or cause bugs in production. One solution to this is to deploy early and update the deployment as the code evolves.
 
-Submit the link to your deployed portfolio site:
+Submit the link to your deployed portfolio site: https://github.com/fmarte23/hey-world-github-deploy
```

Comment:
This is a link to the repo, not a deployed link.  Please provide a link to the live website that your portfolio will be hosted on.